src/mail-app/mail/model/MailModel.ts

```diff
@@ -68,6 +68,7 @@ export const enum LabelState {
 }
 
 export class MailModel {
+	isFolderView: boolean = true
```

Comment:
I am really confused about this property. What is this for? Why is it here?

Reply:
This property has been changed to `selectedMailSetKind: MailSetKind | null = null` but it still serves the same purpose. For now it's checked when showing mail in mail list to show mail folder icons when in label view